### PR TITLE
Tap action page

### PR DESCRIPTION
**Pull request by PureGeekery**
## Proposed change

Move actions into their own page for issue #11886.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: n/a
- This PR fixes or closes issue: #11886

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_lovelace/button.markdown [hunk 1]
```diff
@@ -57,129 +57,22 @@ state_color:
   default: true
 tap_action:
   required: false
-  description: Action to take on tap
+  description: Action taken on card tap. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/button.markdown]:
```suggestion
  description: Action taken on card tap. See [action documentation](/lovelace/actions/#tap-action).
```

File: source/_lovelace/button.markdown [hunk 2]
```diff
@@ -57,129 +57,22 @@ state_color:
...
-      default: none
-    confirmation:
-      required: false
-      description: "Present a confirmation dialog to confirm the action. See `confirmation` object below"
-      type: [boolean, map]
-      default: "false"
 hold_action:
   required: false
-  description: Action to take on tap-and-hold
+  description: Action taken on card tap and hold. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/button.markdown]:
```suggestion
  description: Action taken on card tap and hold. See [action documentation](/lovelace/actions/#hold-action).
```

File: source/_lovelace/button.markdown [hunk 3]
```diff
@@ -57,129 +57,22 @@ state_color:
...
-      default: none
-    confirmation:
-      required: false
-      description: "Present a confirmation dialog to confirm the action. See `confirmation` object below"
-      type: [boolean, map]
-      default: "false"
 double_tap_action:
   required: false
-  description: Action to take on double tap
+  description: Action taken on card double tap. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/button.markdown]:
```suggestion
  description: Action taken on card double tap. See [action documentation](/lovelace/actions/#double-tap-action).
```

File: source/_lovelace/entities.markdown [hunk 4]
```diff
@@ -99,94 +99,16 @@ state_color:
   default: false
 tap_action:
   required: false
-  description: Action to take on tap
+  description: Action taken on card tap. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/entities.markdown]:
```suggestion
  description: Action taken on card tap. See [action documentation](/lovelace/actions/#tap-action).
```

File: source/_lovelace/entities.markdown [hunk 5]
```diff
@@ -99,94 +99,16 @@ state_color:
...
-      default: none
-    service_data:
-      required: false
-      description: "Service data to include (e.g.,  `entity_id: media_player.bedroom`) when `action` defined as `call-service`"
-      type: string
-      default: none
 hold_action:
   required: false
-  description: Action to take on tap-and-hold
+  description: Action taken on card tap and hold. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/entities.markdown]:
```suggestion
  description: Action taken on card tap and hold. See [action documentation](/lovelace/actions/#hold-action).
```

File: source/_lovelace/entities.markdown [hunk 6]
```diff
@@ -99,94 +99,16 @@ state_color:
...
-      default: none
-    service_data:
-      required: false
-      description: "Service data to include (e.g.,  `entity_id: media_player.bedroom`) when `action` defined as `call-service`"
-      type: string
-      default: none
 double_tap_action:
   required: false
-  description: Action to take on double tap
+  description: Action taken on card double tap. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/entities.markdown]:
```suggestion
  description: Action taken on card double tap. See [action documentation](/lovelace/actions/#double-tap-action).
```

File: source/_lovelace/glance.markdown [hunk 7]
```diff
@@ -87,123 +87,16 @@ show_state:
   default: true
 tap_action:
   required: false
-  description: Action to take on tap
+  description: Action taken on card tap. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/glance.markdown]:
```suggestion
  description: Action taken on card tap. See [action documentation](/lovelace/actions/#tap-action).
```

File: source/_lovelace/glance.markdown [hunk 8]
```diff
@@ -87,123 +87,16 @@ show_state:
...
-      default: none
-    confirmation:
-      required: false
-      description: "Present a confirmation dialog to confirm the action. See `confirmation` object below"
-      type: [boolean, map]
-      default: "false"
 double_tap_action:
   required: false
-  description: Action to take on double tap
+  description: Action taken on card double tap. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/glance.markdown]:
```suggestion
  description: Action taken on card double tap. See [action documentation](/lovelace/actions/#double-tap-action).
```

File: source/_lovelace/picture-elements.markdown [hunk 9]
```diff
@@ -64,109 +64,16 @@ title:
   type: string
 tap_action:
   required: false
-  description: Action to take on tap
+  description: Action taken on card tap. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/picture-elements.markdown]:
```suggestion
  description: Action taken on card tap. See [action documentation](/lovelace/actions/#tap-action).
```

File: source/_lovelace/picture-elements.markdown [hunk 10]
```diff
@@ -64,109 +64,16 @@ title:
...
-      default: none
-    confirmation:
-      required: false
-      description: "Present a confirmation dialog to confirm the action. See `confirmation` object below"
-      type: [boolean, map]
-      default: "false"
 hold_action:
   required: false
-  description: Action to take on tap-and-hold
+  description: Action taken on card tap and hold. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/picture-elements.markdown]:
```suggestion
  description: Action taken on card tap and hold. See [action documentation](/lovelace/actions/#hold-action).
```

File: source/_lovelace/picture-elements.markdown [hunk 11]
```diff
@@ -64,109 +64,16 @@ title:
...
-      default: none
-    confirmation:
-      required: false
-      description: "Present a confirmation dialog to confirm the action. See `confirmation` object below"
-      type: [boolean, map]
-      default: "false"
 double_tap_action:
   required: false
-  description: Action to take on double tap
+  description: Action taken on card double tap. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/picture-elements.markdown]:
```suggestion
  description: Action taken on card double tap. See [action documentation](/lovelace/actions/#double-tap-action).
```

File: source/_lovelace/picture-elements.markdown [hunk 12]
```diff
@@ -228,76 +135,18 @@ tap_action:
       description: "Present a confirmation dialog to confirm the action. See `confirmation` object below"
       type: [boolean, map]
       default: "false"
+tap_action:
+  required: false
+  description: Action taken on card tap. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/picture-elements.markdown]:
```suggestion
  description: Action taken on card tap. See [action documentation](/lovelace/actions/#tap-action).
```

File: source/_lovelace/picture-elements.markdown [hunk 13]
```diff
@@ -330,109 +179,16 @@ title:
   type: string
 tap_action:
   required: false
-  description: Action to take on tap
+  description: Action taken on card tap. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/picture-elements.markdown]:
```suggestion
  description: Action taken on card tap. See [action documentation](/lovelace/actions/#tap-action).
```

File: source/_lovelace/picture-elements.markdown [hunk 14]
```diff
@@ -487,109 +243,16 @@ entity:
...
```
Comment (frenck) [inline source/_lovelace/picture-elements.markdown]:
```suggestion
  description: Action taken on card tap. See [action documentation](/lovelace/actions/#tap-action).
```

File: source/_lovelace/picture-elements.markdown [hunk 15]
```diff
@@ -614,109 +277,16 @@ title:
...
```
Comment (frenck) [inline source/_lovelace/picture-elements.markdown]:
```suggestion
  description: Action taken on card tap. See [action documentation](/lovelace/actions/#tap-action).
```

File: source/_lovelace/picture-entity.markdown [hunk 16]
```diff
@@ -65,123 +65,16 @@ theme:
   type: string
 tap_action:
   required: false
-  description: Action to take on tap
+  description: Action taken on card tap. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/picture-entity.markdown]:
```suggestion
  description: Action taken on card tap. See [action documentation](/lovelace/actions/#tap-action).
```

File: source/_lovelace/picture-glance.markdown [hunk 17]
```diff
@@ -69,109 +69,16 @@ theme:
   type: string
 tap_action:
   required: false
-  description: Action to take on tap
+  description: Action taken on card tap. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/picture-glance.markdown]:
```suggestion
  description: Action taken on card tap. See [action documentation](/lovelace/actions/#tap-action).
```

File: source/_lovelace/picture-glance.markdown [hunk 18]
```diff
@@ -194,113 +101,16 @@ show_state:
   default: true
 tap_action:
   required: false
-  description: Action to take on tap
+  description: Action taken on card tap. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/picture-glance.markdown]:
```suggestion
  description: Action taken on card tap. See [action documentation](/lovelace/actions/#tap-action).
```

File: source/_lovelace/picture.markdown [hunk 19]
```diff
@@ -26,123 +26,16 @@ theme:
   type: string
 tap_action:
   required: false
-  description: Action to take on tap
+  description: Action taken on card tap. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/picture.markdown]:
```suggestion
  description: Action taken on card tap. See [action documentation](/lovelace/actions/#tap-action).
```

File: source/lovelace/header-footer.markdown [hunk 20]
```diff
@@ -29,15 +29,15 @@ image:
   type: string
 tap_action:
   required: false
-  description: Action to take on tap.
+  description: Action taken on card tap. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/lovelace/header-footer.markdown]:
```suggestion
  description: Action taken on card tap. See [action documentation](/lovelace/actions/#tap-action).
```

File: source/_lovelace/picture-elements.markdown [hunk 21]
```diff
@@ -228,76 +135,18 @@ tap_action:
...
       type: [boolean, map]
       default: "false"
+tap_action:
+  required: false
+  description: Action taken on card tap. See [action documentation](/lovelace/actions/).
+  type: map
 hold_action:
   required: false
-  description: Action to take on tap-and-hold
+  description: Action taken on card tap and hold. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/picture-elements.markdown]:
```suggestion
  description: Action taken on card tap and hold. See [action documentation](/lovelace/actions/#hold-action).
```

File: source/_lovelace/picture-elements.markdown [hunk 22]
```diff
@@ -330,109 +179,16 @@ title:
...
-      default: none
-    confirmation:
-      required: false
-      description: "Present a confirmation dialog to confirm the action. See `confirmation` object below"
-      type: [boolean, map]
-      default: "false"
 hold_action:
   required: false
-  description: Action to take on tap-and-hold
+  description: Action taken on card tap and hold. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/picture-elements.markdown]:
```suggestion
  description: Action taken on card tap and hold. See [action documentation](/lovelace/actions/#hold-action).
```

File: source/_lovelace/picture-elements.markdown [hunk 23]
```diff
@@ -487,109 +243,16 @@ entity:
...
```
Comment (frenck) [inline source/_lovelace/picture-elements.markdown]:
```suggestion
  description: Action taken on card tap and hold. See [action documentation](/lovelace/actions/#hold-action).
```

File: source/_lovelace/picture-elements.markdown [hunk 24]
```diff
@@ -614,109 +277,16 @@ title:
...
```
Comment (frenck) [inline source/_lovelace/picture-elements.markdown]:
```suggestion
  description: Action taken on card tap and hold. See [action documentation](/lovelace/actions/#hold-action).
```

File: source/_lovelace/picture-entity.markdown [hunk 25]
```diff
@@ -65,123 +65,16 @@ theme:
...
-      default: none
-    confirmation:
-      required: false
-      description: "Present a confirmation dialog to confirm the action. See `confirmation` object below"
-      type: [boolean, map]
-      default: "false"
 hold_action:
   required: false
-  description: Action to take on tap-and-hold
+  description: Action taken on card tap and hold. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/picture-entity.markdown]:
```suggestion
  description: Action taken on card tap and hold. See [action documentation](/lovelace/actions/#hold-action).
```

File: source/_lovelace/picture-glance.markdown [hunk 26]
```diff
@@ -69,109 +69,16 @@ theme:
...
-      default: none
-    confirmation:
-      required: false
-      description: "Present a confirmation dialog to confirm the action. See `confirmation` object below"
-      type: [boolean, map]
-      default: "false"
 hold_action:
   required: false
-  description: Action to take on tap-and-hold
+  description: Action taken on card tap and hold. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/picture-glance.markdown]:
```suggestion
  description: Action taken on card tap and hold. See [action documentation](/lovelace/actions/#hold-action).
```

File: source/_lovelace/picture-glance.markdown [hunk 27]
```diff
@@ -194,113 +101,16 @@ show_state:
...
```
Comment (frenck) [inline source/_lovelace/picture-glance.markdown]:
```suggestion
  description: Action taken on card tap and hold. See [action documentation](/lovelace/actions/#hold-action).
```

File: source/_lovelace/picture.markdown [hunk 28]
```diff
@@ -26,123 +26,16 @@ theme:
...
-      default: none
-    confirmation:
-      required: false
-      description: "Present a confirmation dialog to confirm the action. See `confirmation` object below"
-      type: [boolean, map]
-      default: "false"
 hold_action:
   required: false
-  description: Action to take on tap-and-hold
+  description: Action taken on card tap and hold. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/picture.markdown]:
```suggestion
  description: Action taken on card tap and hold. See [action documentation](/lovelace/actions/#hold-action).
```

File: source/lovelace/header-footer.markdown [hunk 29]
```diff
@@ -29,15 +29,15 @@ image:
...
   type: map
 hold_action:
   required: false
-  description: Action to take on tap-and-hold
+  description: Action to take on tap-and-hold. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/lovelace/header-footer.markdown]:
```suggestion
  description: Action to take on tap-and-hold. See [action documentation](/lovelace/actions/#hold-action).
```

File: source/_lovelace/picture-elements.markdown [hunk 30]
```diff
@@ -228,76 +135,18 @@ tap_action:
...
-      default: none
-    confirmation:
-      required: false
-      description: "Present a confirmation dialog to confirm the action. See `confirmation` object below"
-      type: [boolean, map]
-      default: "false"
 double_tap_action:
   required: false
-  description: Action to take on double tap
+  description: Action taken on card double tap. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/picture-elements.markdown]:
```suggestion
  description: Action taken on card double tap. See [action documentation](/lovelace/actions/#double-tap-action).
```

File: source/_lovelace/picture-elements.markdown [hunk 31]
```diff
@@ -330,109 +179,16 @@ title:
...
```
Comment (frenck) [inline source/_lovelace/picture-elements.markdown]:
```suggestion
  description: Action taken on card double tap. See [action documentation](/lovelace/actions/#double-tap-action).
```

File: source/_lovelace/picture-elements.markdown [hunk 32]
```diff
@@ -487,109 +243,16 @@ entity:
...
```
Comment (frenck) [inline source/_lovelace/picture-elements.markdown]:
```suggestion
  description: Action taken on card double tap. See [action documentation](/lovelace/actions/#double-tap-action).
```

File: source/_lovelace/picture-elements.markdown [hunk 33]
```diff
@@ -614,109 +277,16 @@ title:
...
```
Comment (frenck) [inline source/_lovelace/picture-elements.markdown]:
```suggestion
  description: Action taken on card double tap. See [action documentation](/lovelace/actions/#double-tap-action).
```

File: source/_lovelace/picture-entity.markdown [hunk 34]
```diff
@@ -65,123 +65,16 @@ theme:
...
-      default: none
-    confirmation:
-      required: false
-      description: "Present a confirmation dialog to confirm the action. See `confirmation` object below"
-      type: [boolean, map]
-      default: "false"
 double_tap_action:
   required: false
-  description: Action to take on double tap
+  description: Action taken on card double tap. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/picture-entity.markdown]:
```suggestion
  description: Action taken on card double tap. See [action documentation](/lovelace/actions/#double-tap-action).
```

File: source/_lovelace/picture-glance.markdown [hunk 35]
```diff
@@ -194,113 +101,16 @@ show_state:
...
-      default: none
-    confirmation:
-      required: false
-      description: "Present a confirmation dialog to confirm the action. See `confirmation` object below"
-      type: [boolean, map]
-      default: "false"
 double_tap_action:
   required: false
-  description: Action to take on double tap
+  description: Action taken on card double tap. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/picture-glance.markdown]:
```suggestion
  description: Action taken on card double tap. See [action documentation](/lovelace/actions/#double-tap-action).
```

File: source/_lovelace/picture.markdown [hunk 36]
```diff
@@ -26,123 +26,16 @@ theme:
...
-      default: none
-    confirmation:
-      required: false
-      description: "Present a confirmation dialog to confirm the action. See `confirmation` object below"
-      type: [boolean, map]
-      default: "false"
 double_tap_action:
   required: false
-  description: Action to take on double tap
+  description: Action taken on card double tap. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/_lovelace/picture.markdown]:
```suggestion
  description: Action taken on card double tap. See [action documentation](/lovelace/actions/#double-tap-action).
```

File: source/lovelace/header-footer.markdown [hunk 37]
```diff
@@ -29,15 +29,15 @@ image:
...
   type: map
 double_tap_action:
   required: false
-  description: Action to take on double tap
+  description: Action to take on tap-and-hold. See [action documentation](/lovelace/actions/).
```
Comment (frenck) [inline source/lovelace/header-footer.markdown]:
```suggestion
  description: Action to take on tap-and-hold. See [action documentation](/lovelace/actions/#double-tap-action).
```

File: source/lovelace/actions.markdown [hunk 38]
```diff
@@ -0,0 +1,197 @@
+---
+title: "Actions"
+description: "Define what an object does when interacted with."
+---
+Some Lovelace cards have support for tap actions. These actions define what will happen when you tap or hold on an object within a card.
```
Comment (frenck) [inline source/lovelace/actions.markdown:6]:
```suggestion

Some Lovelace cards have support for tap actions. These actions define what will happen when you tap or hold on an object within a card.
```

File: source/lovelace/actions.markdown [hunk 39]
```diff
@@ -0,0 +1,197 @@
...
+
+Actions can be enabled on:
+
+* {% active_link /lovelace/entities/ Entity %}
+* {% active_link /lovelace/entity-button/ Entity Button %}
+* {% active_link /lovelace/glance/ Glance %}
+* {% active_link /lovelace/picture/ Picture %}
+* {% active_link /lovelace/picture-elements/ Picture Element %}
+* {% active_link /lovelace/picture-entity/ Picture Entity %}
+* {% active_link /lovelace/picture-glance/ Picture Glance %}
```
Comment (frenck) [inline source/lovelace/actions.markdown]:
```suggestion
- [Entity](/lovelace/entities/)
- [Entity Button](/lovelace/entity-button/)
- [Glance](/lovelace/glance/)
- [Picture](/lovelace/picture/)
- [Picture Element](/lovelace/picture-elements/)
- [Picture Entity](/lovelace/picture-entity/)
- [Picture Glance](/lovelace/picture-glance/)
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks, @PurelyNicole! Really nice!

Don't be scared about the amount of review comments/suggestions. They all are more of the same.

I think we can deeplink better by linking to the anchor of the type of action directly. I've added them as review suggestions, so you can accept those if you agree (saves you the effort of changing them all manually).

👍

**Comment by PurelyNicole** [comment]
Agreed, the anchor link is a better approach.

**Review by frenck — APPROVED** [review]
Awesome!

Thanks for doing this! 👍

**Comment by iantrich** [comment]
@PurelyNicole this was so needed! Awesome!
image
wonderful

**Pull request merged**